diff --git a/node_modules/@xstyled/system/dist/index.d.ts b/node_modules/@xstyled/system/dist/index.d.ts
index 92dbdb1..14ad40f 100644
--- a/node_modules/@xstyled/system/dist/index.d.ts
+++ b/node_modules/@xstyled/system/dist/index.d.ts
@@ -45,7 +45,8 @@ declare type ThemeScreens<T extends ITheme> = T extends {
 declare type ThemeStates<T extends ITheme> = T extends {
     states: States;
 } ? T['states'] : unknown;
-declare type ThemeVariants<T extends ITheme> = ThemeScreens<T> & ThemeStates<T>;
+declare type ThemeVariants<T extends ITheme> = ThemeScreens<T> &
+Omit<ThemeStates<T>, '_'>;
 declare type ThemeProp<TType, TTheme extends ITheme> = {
     [P in keyof ThemeVariants<TTheme>]?: TType | ThemeProp<TType, TTheme>;
 };
